canvas_modules/harness/cypress/support/canvas/operation-cmds.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2017-2020 Elyra Authors
```

Comment:
For dates in copyright notices we need to do the following:
1. For brand new files (like this one) we just add the year number. Like this:
``` 
* Copyright 2021 Elyra Authors
```
2. For already existing files, we update the last year number so, for example, this
```
* Copyright 2017-2020 Elyra Authors
```
would change to this:
```
* Copyright 2017-2021 Elyra Authors
```